etc/devtools_parser.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
```

Comment:
Perhaps the existence of this tool could be referenced in the Servo Book too.

Reply:
Yes, it requires at least the cli version of Wireshark. I added some instructions for the most common platforms and a link with more information. Thank you for the catch!

About adding it to the book, I was thinking of writing about the DevTools features once they were mostly working again. Since this is more of a helper to write the structs in the implementation I think we may wait and mention it later with the rest of the implementation. What do you think?

Reply:
Yep, probably makes sense to write up the DevTools all at once.